packages/docs/integrations/channels/smtp.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: Send Email with a SMTP Server
+---
+
+Dittofeed can integrate with basic SMTP servers to send emails. Many third-party email services exist for testing the quality and deliverability of your emails, such as [Mailtrap](https://mailtrap.io/). Using SMTP as an email provider is useful for integrating with these services.
+
+Note that we don't recommend using SMTP for production email delivery. With a plain SMTP server, you'd lose access to to Spam and Delivery rate metrics, which dedicated email service providers are able to provide via their relationships with ISPs.
```

Comment:
delete extra "to", and also not sure if "spam" and "delivery" need capitalization